app/assets/src/components/views/bulk_download/ChooseStep.jsx

```diff
@@ -484,10 +503,26 @@ class ChooseStep extends React.Component {
     );
   };
 
-  render() {
-    const { onContinue, selectedSampleIds } = this.props;
+  renderFilteredSamplesWarning = () => {
+    const { filteredSampleNames } = this.props;
 
-    const numSamples = selectedSampleIds.size;
+    return (
+      <div className={cs.notification}>
+        <Notification type="warn" displayStyle="flat">
+          <span className={cs.filteredSamplesWarning}>
+            {filteredSampleNames.length}{" "}
+            {filteredSampleNames.length > 1 ? "samples " : "sample "}
+            won't be included in this bulk download
+          </span>, because they are in progress or failed samples:{" "}
+          {filteredSampleNames.join(", ")}
```

Comment:
I haven't seen the design but it seems like a bulleted list would be more appropriate here.

Reply:
This is from @happyimadesignr 's design.